src/agent/rustjail/src/container.rs

```diff
@@ -1460,25 +1466,6 @@ lazy_static! {
     };
 }
 
-fn setrlimit(limit: &POSIXRlimit) -> Result<()> {
-    let rl = libc::rlimit {
-        rlim_cur: limit.soft,
-        rlim_max: limit.hard,
-    };
-
-    let res = if RLIMITMAPS.get(limit.r#type.as_str()).is_some() {
```

Comment:
Did the RLIMITMAPS map still needed if we switched to rlimit crate? 
If not, please  remove it too.

Reply:
I'm sorry to forget to remove RLIMITMAPS.
I fixed it.
Thank you.